docs/rules/implicit-arrow-linebreak.md

```diff
@@ -0,0 +1,101 @@
+# Enforce the location of arrow function bodies with implicit returns (implicit-arrow-linebreak)
+
+An arrow function body can contain an implicit return as an expression instead of a block body. It can be useful to enforce a consistent location for the implicitly returned expression.
+
+## Rule Details
+
+This rule aims to enforce a consistent location for an arrow function containing an implicit return.
+
+See Also:
+
+- [`brace-style`](https://eslint.org/docs/rules/brace-style) which enforces this behavior for arrow functions with block bodies.
+
+### Options
+
+This rule accepts a string option:
+
+- `"beside"` (default) disallows a newline before an arrow function body.
+- `"below"` requires a newline before an arrow function body.
+
+Examples of **incorrect** code for this rule with the default `"beside"` option:
+
+```js
+/* eslint arrow-linebreak: ["error", "beside"] */
```

Comment:
The rule has been renamed to `implicit-arrow-linebreak`, but this comment still calls it `arrow-linebreak`. (There are a few other comments like it in this file.)